src/services/SearchService.ts

```diff
@@ -73,7 +78,8 @@ function convertToSearchNodePayload(
 
 async function search<T extends SearchResponseElement>(entity: RawSearchRequestPayload): Promise<T[] | null> {
   try {
-    const response: SearchResponsePayload = (await httpSearch.post({ entity })).data;
+    const newEntity = entity.search ? {...entity, search: convertToPhraseMatch(entity.search)} : entity
```

Comment:
We convert the search before calling the API, but the correct approach is likely calling it upstream, where the query term is parsed.

Reply:
This gives us fine-grained control on which search boxes need to support PhraseMatch. For example, weights do not support Phrase Search and we don't want to convert that.